src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Byte128Vector.java

```diff
@@ -805,6 +805,7 @@ ByteVector fromArray0(byte[] a, int offset) {
         return super.fromArray0Template(a, offset);  // specialize
     }
 
+
```

Comment:
Some files were just modified by adding an empty newline.
Why?

Reply:
It's an unfortunate side-effect of generating files from a template , where in some cases the generation is partial.
In this case generating from `X-VectorBits.java.template` for code specific to the `short` element type.